sentry-ruby/lib/sentry/redis.rb

```diff
@@ -54,7 +54,8 @@ def commands_description
     def parsed_commands
       commands.map do |statement|
         command, key, *arguments = statement
-        command_set = { command: command.to_s.upcase, key: key }
+        command_set = { command: command.to_s.upcase }
+        command_set[:key] = key if Utils::EncodingHelper.valid_utf_8?(key)
```

Comment:
Since `key` is kinda important in commands, do you think it'd be better insert a placeholder like `"<invalid encoded key>"` instead?

Reply:
I think the type of redis command signifies that there's a key there, I think that's fine. 
This is edge-case behaviour anyway.